### DISCUSSION TITLE
Webhook Map fields for creating custom data structure

### DISCUSSION THREAD

**Discussion by lapaDevAdmin** [chatwoot/chatwoot#9876] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

Actually webhook send a standard data, but some thirthparty apps require a specific field name

### Describe the solution you'd like

can you add a settings in webhook where we can maps the field and data we want send?

**Comment by linear** [comment]
<p><a href="https://linear.app/chatwoot/issue/PR-1233/webhook-map-fields-for-creating-custom-data">PR-1233 Webhook Map fields for creating custom data</a></p>

**Comment by sojan-official** [comment]
You can relay the event to a third-party server using a cloud function.

**Comment by mredodos** [comment]
> You can relay the event to a third-party server using a cloud function.

Actually i send the data to make and in make i rebuild the data structure. 
In my opinion, it would be very useful to be able to manipulate the sending of webhook data, also to avoid use of third-party tools and thus additional costs as well as possible malfunctions

**Comment by pranavrajs** [comment]
@mredodos Thanks for the feedback. It is a bit out of scope for us now. We will revisit it later. Moving this to discussions for now.
